doc/ref_arch/kubernetes/chapters/appendix-b.md

```diff
@@ -0,0 +1,41 @@
+[<< Back](../../kubernetes)
+# Appendix B - Guidance For workload isolation(Multitenancy) with Kubernetes for application Vendors
+<p align="right"><img src="../figures/bogo_sdc.png" alt="scope" title="Scope" width="35%"/></p>
+
+## Table of Contents
+
+* [B.1 Overview](#B.1)
+* [B.2 Multitenancy Models for CNFs](#B.2)
+* [B.3 Solution Areas](#B.3)
+
+
+<a name="B.1"></a>
+## B.1 Overview
+
+Problem statement: A single Kubernetes cluster does not provide hard multitenancy by design. Within a Cluster, Kubernetes Namespace is a mechanism to provide Soft isolation multitenancy.
+A Kubernetes Namespace does provide isolation by means of role based access control (RBAC), Resource Isolation and Network Policy, however they are still within the same trust domain and a potential breach of Cluster Admin Role could lead to the Blast Radius across the entire Cluster and all its Kubernetes Namespaces.
+So there is a need to define various use cases or ways to build Multitenancy Deployment Models and define the Best Practices to secure each Model.
+
+<p align="left"><img src="../figures/Model2-cluster-isolation.png" alt="scope" title="Scope" width="50%"/></p> 
```

Comment:
In the diagram headings, can there be a space between "Model" and the number (1 or 2)?

What is the "bus" in both diagrams?